RIGS/signals.py

```diff
@@ -102,3 +105,35 @@ def on_revision_commit(sender, instance, created, **kwargs):
 
 
 post_save.connect(on_revision_commit, sender=models.EventAuthorisation)
+
+
+def send_admin_awaiting_approval_email(user, request, **kwargs):
+    # Bit more controlled than just emailing all superusers
+    for admin in models.Profile.objects.filter(email__in=[y for x in settings.ADMINS for y in x]):
+        # Check we've ever emailed them before and if so, if cooldown has passed.
+        if admin.last_emailed is None or admin.last_emailed + datetime.timedelta(minutes=15) <= timezone.now():
+            context = {
+                'request': request,
+                'link_suffix': '/admin/RIGS/profile/?is_approved__exact=0',
+                'number_of_users': models.Profile.users_awaiting_approval_count(models.Profile.objects),
+                'to_name': admin.first_name
+            }
+
+            email = EmailMultiAlternatives(
+                "%s new users awaiting approval on RIGS" % (context['number_of_users']),
+                get_template("RIGS/admin_awaiting_approval.txt").render(context),
+                to=[admin.email],
+                reply_to=[user.email],
+            )
+            css = staticfiles_storage.path('css/email.css')
```

Comment:
I would recommend pushing out all the sending of the email to a separate function (can still be in this file) so it is abstracted away from the deciding whether to send the email